### DISCUSSION TITLE
Images Won't Load, How Do I Fix This?

### DISCUSSION THREAD

**Discussion by Adm1nu53r** [herndev/ReactJS-Portfolio#12] [Q&A] [answered]
I tried to rebuild the react app but it's giving me an error, it says "image not found"

**Comment by herndev** [comment, marked as answer]
I've removed some of the images in this repository for my personal privacy purposes, for the meantime you can use any images you have or maybe use placeholders to replace some of the lacking images. That might prevent from displaying ``` image not found ```.
